db/migrate/20180301100944_create_work_plans.rb

```diff
@@ -0,0 +1,16 @@
+class CreateWorkPlans < ActiveRecord::Migration[5.1]
+  def change
+    create_table :work_plans, id: :serial do |t|
+      t.integer :project_id
+      t.references :product, foreign_key: true
+      t.string :original_set_uuid
+      t.citext :owner_email, null: false
+      t.string :comment
+      t.date :desired_date
+      t.string :uuid, null: false
```

Comment:
Is storing a `uuid` as well as `id` a deliberate choice? Can this be stored as a `uuid` type?